README.md

```diff
@@ -1,138 +1,187 @@
 # sgrep
 
-[![r2c community slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=for-the-badge&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
+[![CircleCI](https://circleci.com/gh/returntocorp/sgrep.svg?style=svg)](https://circleci.com/gh/returntocorp/sgrep)
+[![r2c Community Slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=flat&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
 
-[sgrep.live](https://sgrep.live/): try it now!
+`sgrep` is a tool for easily detecting and preventing bugs and anti-patterns in
+your codebase. It combines the convenience of `grep` with the correctness of
+syntactical and semantic search. Quickly write rules so you can code with
+confidence.
 
-sgrep, for syntactical \(and occasionnally semantic\) grep, is a tool to help find bugs by specifying code patterns using a familiar syntax. The idea is to mix the convenience of grep with the correctness and precision of a compiler frontend.
+**Try it now:** [https://sgrep.live](https://sgrep.live/)
 
-## Quick Examples
+## Overview
 
-| **pattern** | **will match code like** |
-| :--- | :--- |
-| `$X == $X` | `if (node.id == node.id): ...` |
-| `foo(kwd1=1, kwd2=2, ...)` | `foo(kwd2=2, kwd1=1, kwd3=3)` |
-| `subprocess.Popen(...)` | `import subprocess as s; s.Popen(['foo'])` |
-
-→ [see more examples in the sgrep-rules registry](https://github.com/returntocorp/sgrep-rules)
+Language support:
 
-## Supported Languages
-
-| **javascript** | **python** | **go** | **java** | **c** | **ruby** | **scala** |
+| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Typescript** | **PHP &nbsp; &nbsp;** |
 | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-| ✅ | ✅ | ✅ | ✅ | ✅ | coming | coming |
-
-→ see full language support details in [matrix.md](docs/matrix.md)
+| ✅ | ✅ | ✅ | ✅ | ✅ | Coming... | Coming... |
 
-## Meetups
+Simple patterns for finding code in many languages:
```

Comment:
this makes me think that a single pattern works on multiple languages (which isn't really true). Could we rephrase?